doc/examples/plot_seam_carving.py

```diff
@@ -46,22 +46,22 @@
 
 out = transform.seam_carve(img, eimg, 'vertical', 200)
 plt.figure()
-plt.title('Resized using Seam-Carving')
+plt.title('Resized using Seam Carving')
 plt.imshow(out)
 
 """
 .. image:: PLOT2RST.current_figure
 
-As you can see, resizing as distorted the rocket and the objects around,
-whereas seam carving has reszied by removing the empty spaces in between.
+Resizing distorts the rocket and surrounding objects, whereas seam carving
+removes empty spaces and preserves object proportions.
 
 Object Removal
 --------------
 
-Seam Carving can also be used to remove atrifacts from images. To do that, we
-have to ensure that pixels to be removes get less importance. In the following
-code I approximately mark the rocket with a mask, and then decrease the
-importance of those pixels
+Seam carving can also be used to remove artifacts from images.
+This requires to downweigh the pixels to be removed. In the following
```

Comment:
You're right, this can be improved too. How about something like

"This requires weighting the artifact with low values. Recall lower weights are preferentially removed in seam carving. The following code masks the rocket's region with low weights, indicating it should be removed."